Ktisis/Scene/Objects/SceneObject.cs

```diff
@@ -100,4 +100,15 @@ public IReadOnlyList<SceneObject> GetChildren()
 				yield return reChild;
 		}
 	}
+
+	public bool IsChildOf(SceneObject item) {
```

Comment:
```suggestion
	public bool IsDescendantOf(SceneObject item) {
```

Reply:
`Descendant` here is consistent with the terminology used by Havok, referring to a bone that's affected by another higher in the chain, but is not a direct child of it.